v3/src/components/case-table/collection-table.tsx

```diff
@@ -125,6 +125,12 @@ export const CollectionTable = observer(function CollectionTable(props: IProps)
       caseTableModel)
   }, [caseTableModel, columns, forceUpdate])
 
+  useEffect(() => {
+    if (collectionTableModel?.attrIdToEdit !== undefined) {
+      gridRef.current?.selectCell({idx: columns.length, rowIdx: -1})
```

Comment:
It would be better to figure out the index based on the id, rather than assuming it's the right-most column. That's a safe assumption right now but it could change in the future, in which case this would cause an obscure bug to solve.